config.py

```diff
@@ -230,6 +216,30 @@ def data_directory(cls):
         return cls.get(cls.DATA_DIRECTORY)
 
     @classmethod
+    def load_cdns(cls, _db, config_instance=None):
+        from model import ExternalIntegration as EI
+        CDN_GOAL_KEY = {
+            EI.BOOK_COVERS_GOAL : 'book_covers',
+            EI.OA_CONTENT_GOAL : 'open_access_books',
+            EI.OPDS_FEED_GOAL : 'opds'
+        }
+
+        cdns = _db.query(EI).filter(EI.protocol==EI.CDN).all()
+        if not cdns:
+            return
+
+        cdn_integration = dict()
+        for cdn in cdns:
+            netloc = CDN_GOAL_KEY.get(cdn.goal)
+            if not netloc:
+                continue
+
+            cdn_integration[netloc] = cdn.url
```

Comment:
I don't get this part. `netloc` is a urllib2 term for a domain name, but here the "netloc" would be "book_covers", "open_access_books", etc.